media-gfx/gifsicle/gifsicle-1.91.recipe

```diff
@@ -41,12 +42,17 @@ PATCH()
 BUILD()
 {
 	runConfigure ./configure
+	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+		# avoid undefined reference to __sync_add_and_fetch
+		# from function scale_image_threaded in src/xform.c
+		sed -i -e "/HAVE___SYNC_ADD_AND_FETCH/ s/1/0/" config.h
+	fi
 	make
 }
 
 INSTALL()
 {
-	make install
+	make install-strip
```

Comment:
Shouldn't we use `defineDebugInfoPackage` instead of dropping debuginfo by stripping?

Reply:
Yep, I've been thinking about this, but I did not dare to add a debuginfo package because the package does not have any library.
But I can think of a good reason to add it: it makes it (slightly) easier to for a user to investigate possible bugs, as the user would just need to install the debuginfo package instead of having to rebuild everything.

So I'll add the debuginfo package right now ;)

Reply:
Tagging #2135, as we still don't have an official consensus on what should be done with debug info in binaries